tests/rkt_tests.go

```diff
@@ -628,9 +630,7 @@ func serverHandler(t *testing.T, server *taas.Server) {
 	}
 }
 
-func runSignImage(t *testing.T, imageFile string, keyIndex int) string {
-	ascFile := fmt.Sprintf("%s.asc", imageFile)
-
+func runSignImage(t *testing.T, imagePath string, keyIndex int) string {
 	// keys stored in tests/secring.gpg, tests/pubring.gpg, tests/key1.gpg, tests/key2.gpg
```

Comment:
I think that `pubring.gpg` was never needed, while `key1.pgp` and `key2.pgp` are used by runRktTrust.

Will remove the comment since it's misleading.